s3prl/downstream/a2o-vc-vcc2020/README.md

```diff
@@ -0,0 +1,92 @@
+# S3PRL-VC: open-source voice conversion framework with self-supervised speech representations
+
+Development: [Wen-Chin Huang](https://github.com/unilight) @ Nagoya University (2021).  
+If you have any questions, please open an issue, or contact through email: wen.chinhuang@g.sp.m.is.nagoya-u.ac.jp
+
+## Task
+
+In this downstream, we focus on training any-to-one (A2O) voice conversion (VC) models on the two tasks in [voice conversion challenge 2020 (VCC2020)](https://github.com/nii-yamagishilab/VCC2020-database).
+The first task is intra-lingual VC, and the second task is cross-lingual VC.
+For more details, please refer to the original paper:
+[Yi, Z., Huang, W., Tian, X., Yamagishi, J., Das, R.K., Kinnunen, T., Ling, Z., Toda, T. (2020) Voice Conversion Challenge 2020 –- Intra-lingual semi-parallel and cross-lingual voice conversion –-. Proc. Joint Workshop for the Blizzard Challenge and Voice Conversion Challenge 2020, 80-98, DOI: 10.21437/VCC_BC.2020-14.](https://www.isca-speech.org/archive_v0/VCC_BC_2020/pdfs/VCC2020_paper_13.pdf)
+
+## Implementation
+
+We implement three models. Two of them resemble the the top systems in VCC2018 and VCC2020, as described in the following papers:
+[Liu, L., Ling, Z., Jiang, Y., Zhou, M., Dai, L. (2018) WaveNet Vocoder with Limited Training Data for Voice Conversion. Proc. Interspeech 2018, 1983-1987, DOI: 10.21437/Interspeech.2018-1190.](https://www.isca-speech.org/archive/Interspeech_2018/pdfs/1190.pdf)  
+[Liu, L., Chen, Y., Zhang, J., Jiang, Y., Hu, Y., Ling, Z., Dai, L. (2020) Non-Parallel Voice Conversion with Autoregressive Conversion Model and Duration Adjustment. Proc. Joint Workshop for the Blizzard Challenge and Voice Conversion Challenge 2020, 126-130, DOI: 10.21437/VCC_BC.2020-17.](https://www.isca-speech.org/archive_v0/VCC_BC_2020/pdfs/VCC2020_paper_17.pdf)
+
+We made several modifications.
+1. **Input**: instead of using the bottleneck features (BNFs) of a pretrained ASR model, we use the various upstreams provided in S3PRL.
+2. **Output**: instead of using acoustic features extracted using a high-quality vocoder, STRAIGHT, we use the log-melspectrograms.
+3. **Data**: we benchmark on the [VCC2020](https://github.com/nii-yamagishilab/VCC2020-database) dataset. 
+4. **Training strategy**: instead of pretraining on a multispeaker dataset first, we directly trained on the target speaker training set.
+5. **Vocoder**: instead of using the WaveNet vocoder, we used the [Parallel WaveGAN](https://arxiv.org/abs/1910.11480) (PWG) based on the [open source project](https://github.com/kan-bayashi/ParallelWaveGAN) developed by [kan-bayashi](https://github.com/kan-bayashi).
+
+## Dependencies:
+
+- `parallel-wavegan`
+- `fastdtw`
+- `pyworld`
+- `pysptk`
+- `jiwer`
+- `resemblyzer`
+
+## Usage
+
+### Preparation
+```
+# Download the VCC2020 dataset.
+cd <root-to-s3prl>/s3prl/downstream/a2o-vc-vcc2020
+cd data
+./data_download.sh vcc2020/
+cd ../
+
+# Download the pretrained PWGs.
+./pwg_download.sh ./
```

Comment:
It seems the pwd_download.sh was not added into git tracking XD

Reply:
My bad... I'll add it.

Reply:
Ah, I found that I changed the file name. It should be `vocoder_download.sh` now. I'll change the README.

Reply:
If you did not have the PWGs, how did you test the evaluation scripts? 😮

Reply:
No! I not yet test the evaluate or decode scripts. What I meant by running through the train/evaluation is actually just the training/evaluation loop defined in the expert.py :) In your case the evaluation loop simply dump the files for future evaluation so I not yet touch the vocoder and metric computation parts